CHANGELOG.txt

```diff
@@ -129,6 +129,10 @@ Changes:
    * Add support for custom tags (see #2284).
  - obspy.io.sh:
    * fix appending traces to existing Q file (see #2870)
+ - obspy.io.sac:
+   * expose a `keep_sac_header` flag to the `write` plugin, which allows users
+     to write a "fresh" SAC header, instead of attempting to merge information
+     from an old SAC header, if one exists. (see #2760)
```

Comment:
Oh, actually, can you please rebase one last time on current master and move that bullet all the way up to "master" changes? =)